descriptions.md

```diff
@@ -592,6 +592,21 @@ Describes the possible actions members in the presence set can emit.
 | LEAVE | TP2 | A member who was present has now left the channel. This may be a result of an explicit request to leave or implicitly when detaching from the channel. Alternatively, if a member's connection is abruptly disconnected and they do not resume their connection within a minute, Ably treats this as a leave event as the client is no longer present. |
 | UPDATE | TP2 | An already present member has updated their member data. Being notified of member data updates can be very useful, for example, it can be used to update the status of a user when they are typing a message. |
 
+
+## enum MessageAction
+
+Describes the possible actions message can represent (in order from zero).
```

Comment:
Where did these descriptions come from? It's not very clear to me from looking at these how a user of the library is meant to use these actions or the two new `Message` fields.